document/CoreFormat.mdk

```diff
@@ -1743,13 +1729,138 @@ A `codeFlow` object may contain a property named `tags`, whose value is a tag se
+
+A `stackFrame` object may contain a property named `message` whose value is a string
+containing a message relevant to this stack frame.
+
+### `uri` property { #stackFrame-uri }
+
+A `stackFrame` object may contain a property named `uri` whose value is a string
+containing the URI of the source code file to which this stack frame refers.
+
+~ IsoNote
+
+The `uri` property will be absent if the analysis tool does not have information
+that maps execution addresses to source code files.
+~
+
+### `line` property { #stackFrame-line }
+
+A `stackFrame` object may contain a property named `line` whose value is an integer
+containing the line number within the file specified by `uri` (&sect;[#stackFrame-uri])
+to which this stack frame refers.
+
+If the `uri` property is absent, the `line` property shall be absent.
+
+~ IsoNote
+The `line` and `column` (&sect;[#stackFrame-column]) properties will be absent if the
+analysis tool does not have information that maps execution addresses to source code lines.
+~
+
+### `column` property { #stackFrame-column }
+
+A `stackFrame` object may contain a property named `column` whose value is an integer
+representing the column number within the line specified by `line` (&sect;[#stackFrame-line])
+to which this stack frame refers.
+
+If the `line` property is absent, the `column` property shall be absent.
+
+### `module` property { #stackFrame-module }
+
+A `stackFrame` object may contain a property named `module` whose value is a string
+containing the name of the module that contains the location to which this stack frame refers.
+
+### `fullyQualifiedLogicalName` property { #stackFrame-fullyQualifiedLogicalName }
+
+A `stackFrame` object may contain a property named `fullyQualifiedLogicalName` whose value
+is a string containing the fully qualified name of the method to which this stack frame refers.
+See &sect;[#location-fullyQualifiedLogicalName] for examples.
+
+~ IsoNote
+The `fullyQualifiedLogicalName` property will be absent in the analysis tool does not
+have symbol information for this stack frame, for example, if this stack frame represents
+a call into an internal operating system function.
+~
+
+### `logicalLocationKey` property { #stackFrame-logicalLocationKey }
+
+A `stackFrame` object may contain a property named `logicalLocationKey` whose value is a string.
+If present, this string shall be equal to the name of one of the properties on the `run.logicalLocations` object (&sect;[#run-logicalLocations]),
+which provides additional information about the logical location specified by `fullyQualifiedLogicalName` (&sect;[#stackFrame-fullyQualifiedLogicalName]).
+For more information about the purpose of this property, see &sect;[#location-fullyQualifiedLogicalName].
+
+### `address` { #stackFrame-address }
+
+A `stackFrame` object may contain a property named `address` whose value is an integer
+containing the byte offset of the location represented by this stack frame from the start
```

Comment:
I'm not sure this is right. Typically a stack frame address is the actual address of the function (+ offset ), not rendered as the offset from base address of the module. This reflects the literal execution of the code (where the return address for execution is pushed onto the stack